ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -110,6 +139,8 @@
         }
         function onInit(e) {
             let formContainer =  e.detail;
+            // creating hamburger menu
+            hamburgerMenuInstance.initializeHamburgerMenu(formContainer);
```

Comment:
You don't need this, once hamburgerInstance is part of form container, you can remove this.